### ISSUE TITLE
Purify 2B in new kernels

### ISSUE THREAD

**Issue by cortner** [ACEsuit/ACEpotentials.jl#228]
@CheukHinHoJerry  -- I think before we can get rid of ACE1 and ACE1x we may want to implement at least the 2B purification in the new implementation. Do you have any thoughts on that? Can we both think about it and discuss early next week?

**Comment by CheukHinHoJerry** [comment]
Sure - I will mull over this during weekend.

**Comment by cortner** [comment]
@CheukHinHoJerry  -- I think this is the only missing feature for 0.8. Have you managed to give any thought to this yet? Don't worry if not, but I don't want to start if you have already done something.

**Comment by CheukHinHoJerry** [comment]
I haven't done anything on this yet. We discussed last time that we are more inclined to do this as a separate lux layer. I am still uncertain on how to do this efficiently but maybe we shouldn't worry about the cost and just do this naively?

**Comment by cortner** [comment]
Let me see if I find any time to have a first go at this over the next two days.

**Comment by cortner** [comment]
just to record here that there is a draft in P4ML, but it seems to be a little non-trivial to make this both efficient and general (i.e. non-polynomials...)
